Analysis/Tutorials/src/histograms.cxx

```diff
@@ -35,11 +35,17 @@ struct ATask {
 
 struct BTask {
   OutputObj<TH2F> etaphiH{TH2F("etaphi", "etaphi", 100, 0., 2. * M_PI, 102, -2.01, 2.01)};
+  // Create a configurable which can be used inside the process method.
+  Configurable<float> phiCut{"phiCut", 6.29f, "A cut on phi"};
```

Comment:
@aalkin do you already have some sort of scalar node for the filter? I was thinking that maybe should allow to have also:

```
Configurable<float> phiCut{"phiCut", 6.29f, "A cut on phi"};
Filter mySimplyPhiCut = track::phi < phiCut;
```

besides

```
Filter mySimplyPhiCut = track::phi < configurable("phiCut", 6.28, "A cut on Phi");
```

so that we can reuse Configurables across multiple Filters.

Reply:
There is already expressions::LiteralNode that can be used for that purpose. Using the term "cut" bot for the value and for the filter may lead to confusion, though. Currently you can do:
`auto cut = expressions::LiteralNode{6.29f}`

Reply:
mmm, not sure. literal node does not have a binding, does it?